build.sbt

```diff
@@ -74,6 +74,7 @@ lazy val core = (project in file("core")).
     name := "gigahorse-core",
     libraryDependencies ++= Seq(sslConfig, reactiveStreams, slf4jApi, scalatest % Test),
     managedSourceDirectories in Compile += (sourceDirectory in Compile).value / "contraband-scala",
+    unmanagedSourceDirectories in Compile += (sourceDirectory in Compile).value / "contraband-scala",
```

Comment:
It's for the inclusion of managed source, so maybe a better solution might be to figure out where the source jars are made and fix the mapping.